Solutions/CohesitySecurity/Tools/grant_keyvault_permission.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/zsh
+SCRIPT=$(realpath "$0")
+SCRIPTPATH=$(dirname "$SCRIPT")
+cd "$SCRIPTPATH"
+
+# Description: This script grants "Get" permissions to the specified user and playbooks on the Key Vault.
+
+# Set variables
+keyvault_prefix="cohesitypro"
+playbook_names=("Cohesity_Close_Helios_Incident" "Cohesity_CreateOrUpdate_ServiceNow_Incident" "Cohesity_Delete_Incident_Blobs" "Cohesity_Restore_From_Last_Snapshot" "Cohesity_Send_Incident_Email")
```

Comment:
Key vault permission should only be necessary for Cohesity_Close_Helios_Incident and Cohesity_Restore_From_Last_Snapshot

Reply:
done